.github/workflows/always.yml

```diff
@@ -65,23 +66,23 @@ jobs:
       fail-fast: false
       max-parallel: 6
       matrix:
-        dotnet: ['3.X', '5.X']
+        dotnet-version: ['3.X', '6.X']
```

Comment:
Is this really any good?

I based it on [this article](https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-net), however it would seem more beneficial to try to force the target framework in itself to see if its forward compatible